README.md

```diff
@@ -1,25 +1,31 @@
 # Just in time
 
-Just in time provided a data visualisation project for DUNE exactly when it was needed. Although it worked, it was a small scaled project with no expantion capabilities, and an expandable version of the project was by now long overdue. "It's about time" aims to provide an implementation of the Just in time functionality with code that's easily expandable and much more robust file structure.
+Just in time provided a data visualisation project for DUNE exactly when it was needed. Although it worked, it was a small scaled project with no expansion capabilities, and an expandable version of the project was by now long overdue. "It's about time" aims to provide an implementation of the Just in time functionality with code that's easily expandable and much more robust file structure.
 
 ## Quick start
 
 ### Dev area setup
-```sh
-# Create a  DBT work area witl local python environment
-dbt-create -c <release> <workarea>
-cd <workarea>
+To use `justintime`, you should first setup up a DUNE DAQ environment. You can follow the instructions in the [DUNE DAQ wiki](https://dune-daq-sw.readthedocs.io/en/latest/packages/daq-buildtools/) to do so. For convenience, the steps below will set up a local environment based on a nightly build of DUNE DAQ. These steps are tested and working as of March 2024, but in case they become outdated, please consult the aforementioned wiki link for the most up-to-date information.
+```
+source /cvmfs/dunedaq.opensciencegrid.org/setup_dunedaq.sh
```

Comment:
Tis is a repeat of basic daq area instructions and should be avoided. 
A link to basic setup instructions should be used instead.